### ISSUE TITLE
Upload,  custom handler

### ISSUE THREAD

**Issue by sadortun** [lusaxweb/vuesax#321]
**Is your feature request related to a problem? Please describe.**
When uploading files, need to upload the file to a custom storage service ( S3, Firebase, Google Storage,  etc)

**Describe the solution you'd like**
It would be nice to have the possibility to add a custom callback function that would perform the upload 

**Describe alternatives you've considered**
This would allow to use the upload component for any kind of back-end, and prevent having to use 3rd party upload plugins.

**Comment by sadortun** [comment]
@luisDanielRoviraContreras how do you se this implementation ?

I guess one of the callback argument will be a function that will need to be called by the user to complete the upload?

**Comment by luisDanielRoviraContreras** [comment]
Do you mean something like `@ upload` and how parameters return the files to send?

**Comment by sadortun** [comment]
@luisDanielRoviraContreras  I added an example in the description above

**Comment by rbpaul2** [comment]
I think the 'action' event prop should be called 'on-change' to follow the conventions of the standard HTML file input element

**Comment by faci** [comment]
Hello ! I have the same problem here. Did you find a solution ?

**Comment by sadortun** [comment]
I am not using that component.

**Comment by faci** [comment]
Ok ok i found the solution anyway :) but thanks for the answer !

**Comment by sadortun** [comment]
@faci  I really would have hope for an update on this, but as you see, we are almost 2 years later :) I had to do some digging to figure out on what project it was used :) 

Have a nice day!

**Comment by faci** [comment]
@sadortun  Yes no problem I was desperate when I saw 2018 But I found a pretty good solution I think 

Have a nice day too !
